shared/AppInsightsCore/src/JavaScriptSDK/CoreUtils.ts

```diff
@@ -113,6 +113,43 @@ function _detachEvent(obj: any, eventNameWithoutOn: string, handlerRef: any, use
     }
 }
 
+/**
+ * Try to define get/set object property accessors for the target object/prototype, this will provide compatibility with
+ * existing API definition when run within an ES5+ container that supports accessors but still enable the code to be loaded
+ * and executed in an ES3 container, providing basic IE8 compatibility.
+ * @param target The object on which to define the property.
+ * @param prop The name of the property to be defined or modified.
+ * @param getProp The getter function to wire against the getter.
+ * @param setProp The setter function to wire against the setter.
+ * @returns True if it was able to create the accessors otherwise false
+ */
+export function objDefineAccessors<T>(target: any, prop: string, getProp?: () => T, setProp?: (v: T) => void): boolean {
```

Comment:
Moved from static CoreUtils implementation (for minification), but still referenced from CoreUtils for backward compatibility